docs/api-style-guide.md

```diff
@@ -0,0 +1,11 @@
+# GruppOne's API style guide
+
+<!-- TODO write this docs file! -->
+<!-- XXX should this be in ita or en? -->
```

Comment:
Direi in italiano. Una regola potrebbe essere che va tutto in inglese tranne la cartella docs. In questo modo la documentazione che ci mettiamo dentro poi può essere riciclata per i manuali.

Reply:
e per quanto riguarda la "documentazione" sotto forma di commenti nel codice? tipo, nel docker-compose ho messo un milione di cose. quello secondo me tanto vale farlo in ita. E il readme dello script che fa le release è fuori da docs.

altra regola potrebbe essere tutti e soli i file *.md in ita, il resto eng

Reply:
> altra regola potrebbe essere tutti e soli i file *.md in ita, il resto eng

Ci sta

> e per quanto riguarda la "documentazione" sotto forma di commenti nel codice?

Quello direi in inglese. Passare dall'italiano all'inglese e viceversa continuamente dà un sacco fastidio (almeno a me), sia finchè sto scrivendo sia finchè sto leggendo.

Reply:
perfetto. segno come risolto e lo scrivo sul gruppo